CETCompression.md

```diff
@@ -0,0 +1,379 @@
+# Contract Execution Transaction Compression
+
+## Introduction
+
+When constructing a DLC for a [numeric outcome](NumericOutcome.md), there are often an unreasonably large number of
+possible outcomes to construct a unique CET for every outcome.
+We remedy this fact with a CET compression mechanism specified in this document which allows
+any flat portions of the DLC's [payout curve](PayoutCurve.md) to be covered with only a logarithmic number of CETs.
+
+It is common for payout curves to have constant extremal payouts for a large number of cases
+representing all outcomes considered sufficiently unlikely.
+These intervals with constant extremal payouts are often called "collars" and these collars
+can be compressed to negligible size making the remaining number of CETs proportional
+to the number of sufficiently likely outcomes.
+Furthermore, through the use of [rounding intervals](NumericOutcome.md#rounding-intervals), even portions of the payout curve which are not
+completely flat can be compressed to some extent, normally causing the total number of CETs to be
+divided by some power of two.
+
+This is accomplished through the use of digit decomposition where oracles attesting to
+numeric outcomes sign each digit of the outcome individually.
+There are as many nonces as there are possible digits required and CETs are claimed using
+only some of these signatures, not necessarily all of them.
```

Comment:
Maybe mention primairly using base 2 here

Reply:
I'm not sure this is the best place for this